src/logic/operate.js

```diff
@@ -1,35 +1,40 @@
 import Big from 'big.js';
 
 const operate = (numberOne, numberTwo, operation) => {
-  const bigNumberOne = new Big(numberOne);
-  const bigNumberTwo = new Big(numberTwo);
+  let intOne;
+  let intTwo;
   let total;
+  if (numberOne) {
+    const bigNumberOne = new Big(numberOne);
+    intOne = parseInt(bigNumberOne, 10);
+  }
+  if (numberTwo) {
+    const bigNumberTwo = new Big(numberTwo);
+    intTwo = parseInt(bigNumberTwo, 10);
+  }
+
   switch (operation) {
     case '+':
-      total = bigNumberOne + bigNumberTwo;
+      total = intOne + intTwo;
       break;
     case '−':
-      total = bigNumberOne - bigNumberTwo;
+      total = intOne - intTwo;
       break;
     case '÷':
-      total = bigNumberOne / bigNumberTwo;
+      total = intOne / intTwo;
       break;
     case '×':
-      total = bigNumberOne * bigNumberTwo;
-      break;
-    case '+/-':
-      total = numberTwo ? bigNumberTwo * -1 : bigNumberOne * -1;
+      total = intOne * intTwo;
```

Comment:
### Solve for the `x` (times) operator functionality
- [ ] The `x` (times) operator functionality is currently down (not functional) Let's work on it please, thank you.
![x-operator](https://user-images.githubusercontent.com/22345961/108001802-91988400-6ffe-11eb-912f-315f9cb8361e.gif)